addons/test_discuss_full/tests/test_performance.py

```diff
@@ -9,18 +9,14 @@
 
 @tagged('post_install', '-at_install')
 class TestDiscussFullPerformance(HttpCase):
-    _query_count = 66
+    _query_count = 69
```

Comment:
Query count should be increased to `68` (+2), not `69`.
I think there was an issue at some point on runbot requiring an extra query count, but this has been fixed apparently